package.json

```diff
@@ -1,36 +1,36 @@
 {
   "name": "start-ui-website",
-  "version": "0.1.0",
-  "private": true,
+  "type": "module",
+  "version": "0.0.1",
   "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint",
-    "pretty": "prettier -w ."
+    "dev": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro"
   },
   "dependencies": {
+    "@astrojs/mdx": "4.3.7",
+    "@astrojs/react": "4.4.0",
```

Comment:
I think we don't need React on this project at the moment, Astro files only is fine.